.github/scripts/deploy.sh

```diff
@@ -0,0 +1,16 @@
+git config --global user.email digitalpreservation@nationalarchives.gov.uk
```

Comment:
Is that the correct email address? Do you know who monitors/has access to this email address?

Reply:
I copied it off the [Jenkins lib repo ](https://github.com/nationalarchives/tdr-jenkinslib/blob/master/vars/tdr.groovy#L78) so I guess someone is watching it. I imagine David has access.